index.js

```diff
@@ -40,9 +38,9 @@ export default async function getStream(inputStream, options = {}) {
 
 		return getBufferedValue();
 	} catch (error) {
-		if (length <= BufferConstants.MAX_LENGTH) {
+		try {
 			error.bufferedData = getBufferedValue();
-		}
+		} catch {}
```

Comment:
We could use `BufferConstants.MAX_STRING_LENGTH`, but using a `try`/`catch` instead removes the Node.js-specific part of the logic, and is simpler. The `getBufferedValue()` function is very small, so it should throw only exceptions related to big inputs.